scripts/deployDocs.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# package docs and deploy
+
+DOC_PATH='./docs'
+VERSION=`cat $DOC_PATH/source/conf.py | egrep 'release\s?=' | awk '{print $3}' | sed -e "s/^'//" -e "s/'$//"`
+PROJECT_NAME=`cat $DOC_PATH/source/conf.py | egrep 'project\s?=' | awk '{print $3}' | sed -e "s/^'//" -e "s/'$//"`
+SOURCE_FILES="$DOC_PATH/build/html/"
+PACKAGE_NAME="docs_${PROJECT_NAME}_${VERSION}"
+DEPLOY_FILENAME="${DOC_PATH}/${PACKAGE_NAME}.tar.gz"
+
+DEPLOY_USER=docs_deploy
+
+if [ ! -z "$1" ]; then
+    DEPLOY_SERVER="$1"
+fi
+
+if [ ! -z "$2" ]; then
+    DEPLOY_USER="$2"
+fi
+
+echo "building docs package $PACKAGE_NAME"
+
+# install dev packages for doc build
+pip install -e .[dev] -U tox-travis
+
+# make the docs from source
+make docs
+
+# package into a tar.gz file for deployment
+tar -czvf "$DEPLOY_FILENAME" "$SOURCE_FILES"
+
+if [ ! -z "$DEPLOY_SERVER" ]; then
+    DEPLOY_BUILD_URL="http://${DEPLOY_SERVER}/docs_build"
+
+    echo "Deploying doc file to $DEPLOY_SERVER"
+    openssl aes-256-cbc -K $encrypted_e14c59e0af38_key -iv $encrypted_e14c59e0af38_iv \
+    -in docs/keys/dex-docs-deploy.enc \
```

Comment:
Presumably docs/keys/dex-docs-deploy.enc is a private SSH key that corresponds to a pubkey authorized for the docs user on shrimp, correct? If so where can we find the private (e.g. id_rsa) and public (id_rsa.pub) parts of that key (along with the passphrase, if set) for that SSH key?

Reply:
I have the public/private, keys I can add them to the root of shrimp with 0600 file access. The file 'docs/keys/dex-docs-deploy.enc' is the private key encrypted by travis to access the account in shrimp.